data/core/faqs.md

```diff
@@ -146,410 +141,6 @@
     - action_deactivate_form
     - form{"name": null}
 
-## new to rasa + not new to chatbots + not migrating
```

Comment:
I don't think all of these should be deleted but rather changed in the same way as in the chitchat file. We should be able to handle faq at any point in this initial flow

Reply:
I will open this as a new issue.